packages/data-explorer/src/views/MainView.vue

```diff
@@ -87,23 +81,9 @@ export default Vue.extend({
         this.setToasts(value)
       }
     },
-    ...mapState([
-      'filters',
-      'showSelected',
-      'dataDisplayLayout',
-      'tableData',
-      'tableName',
-      'tableMeta',
-      'tableSettings',
-      'searchText'
-    ]),
-    ...mapState('header', [
-      'breadcrumbs'
-    ]),
-    ...mapGetters([
-      'isUserAuthenticated',
-      'compressedRouteFilter'
-    ]),
+    ...mapState(['filters', 'showSelected', 'dataDisplayLayout', 'tableData', 'tableName', 'tableMeta', 'tableSettings', 'searchText', 'loading']),
+    ...mapState('header', ['breadcrumbs']),
+    ...mapGetters(['isUserAuthenticated', 'compressedRouteFilter']),
```

Comment:
can we keep the long format , is easer to scan

Reply:
- Ran linter -- fix
- A quick look tough the files tels me that most files have 1 liners like this. if there is a wish to make mayor coding changes lets put it in a separate PR ( for now i think its more important to have a consistent styling over all files )